### PR TITLE
users should see a list of all articles and added minimal styling

### PR DESCRIPTION
**Pull request by zahraakarim**

### CODE REVIEW

File: src/Components/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  const handleLogin = () => {
+    let path = `/articles`;
+    navigate(path);
+  };
+  return (
+    <div id="login">
+      <form action="#!" id="main">
+        <h2>Login to your account</h2>
+
+        <div class="input-parent">
```
Comment (robert112233) [inline src/Components/Home.jsx]: The word class is causing an error, please resolve all warnings/errors in the terminal 👍

File: src/Components/Article-List.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          <div className="article-card">
+            <li key={article.article_id}>
+              <h3 className="article-title"> {article.title}</h3>
+              <h4>Topic: {capitalisedTopic}</h4>
+              <img
+                src={article.article_img_url}
+                alt={`This is an image for ${article.title}`}
+              ></img>
+              <p>Author: {article.author}</p>
+              <p>Posted: {article.created_at}</p>
```
Comment (robert112233) [inline src/Components/Article-List.jsx]: Created at is currently in a non-human-readable format from SQL, you can use a utility function to format this nicely. 

`const dateFormatter = (created_at) => {return new Date(created_at).toLocaleDateString()}`

**Review by robert112233 — CHANGES_REQUESTED** [review]
Great work Zahraa! I really like the dark mode and the login looks very professional. There are a few changes you can make to improve. I would also add an isLoading for your articles list, as I was waiting a few seconds and I thought there was an error. Please implement these changes and submit another PR for review. Thanks!

File: src/Components/Article-List.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import { dateFormatter } from "../util";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getAllArticles().then((data) => {
+      setIsLoading(false);
+      setArticles(data);
```
Comment (yalicee) [inline src/Components/Article-List.jsx]: Flip these around so it stops loading after you setArticles

**Review by yalicee — APPROVED** [review]
Approved :)

**Pull request merged**